src/components/OSCALControl.js

```diff
@@ -61,6 +61,21 @@ export default function OSCALControl(props) {
     );
   }
 
+  // When FedRAMP, retrieve the set-parameters from the initial implementation request statement
```

Comment:
This isn't necessarily specific to NIST or FedRAMP any profile could be defined this way, and `implReqStatements` is actually short for 'implemented-requirements statements'.

Reply:
I had been working with mostly NIST profiles, and assumed FedRAMP had there own method of laying out this information. Thanks for the clarification. I'll change this to something like "Retrieve the set-parameters when contained in the initial implemented requirement statement"